### PR TITLE
AIRNEIS-57-see-the-header-on-mobile

### PR DESCRIPTION
**Pull request by hallycsse**
image

### CODE REVIEW

**Review by BarnabePILLIAUDIN — APPROVED** [review]
Code is good. Same for IOS tests

**Comment by BarnabePILLIAUDIN** [comment]
> LGTM! For code and IOS tests

File: mobile/mobileConfig.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    lgIcon: {
+      size: 30,
+      strokeWidth: 2,
+      color: "#FFEDE6"
+    },
+    xlIcon: {
+      size: 33,
+      strokeWidth: 2,
+      color: "#FFEDE6"
+    }
```
Comment (avetisk) [inline mobile/mobileConfig.js]: Same issue as with @BarnabePILLIAUDIN: please refrain from using different files as source of truth: better keep those values where they belong to: a dedicated file for the design, à la tailwind.config.js

File: mobile/App.js [hunk 2]
```diff
@@ -1,10 +1,6 @@
 import React from "react"
-import { Text, View } from "react-native"
+import Page from "./src/components/Page"
 
-const App = () => (
-  <View className="flex-1 items-center justify-center bg-white">
-    <Text className="text-blue-600">Hello World</Text>
-  </View>
-)
+const App = () => <Page></Page>
```
Comment (avetisk) [inline mobile/App.js]: `<Page />`

File: mobile/src/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { Text, View } from "react-native"
+import { MagnifyingGlassIcon, Bars3Icon, ShoppingCartIcon } from "react-native-heroicons/outline"
+import ButtonWithSvg from "./ButtonWithSvg"
+import mobileConfig from "../../mobileConfig.js"
+
+const Header = () => (
+  <View className="bg-[#382E2B] py-1 flex items-center flex-row justify-between px-4">
```
Comment (avetisk) [inline mobile/src/components/Header.jsx]: No hard magic values!

File: mobile/src/components/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { Text, View } from "react-native"
+import { MagnifyingGlassIcon, Bars3Icon, ShoppingCartIcon } from "react-native-heroicons/outline"
+import ButtonWithSvg from "./ButtonWithSvg"
+import mobileConfig from "../../mobileConfig"
+import { colors } from "../styles"
+
+const Header = () => (
+  <View className="bg-brown-100 py-1 flex items-center flex-row justify-between px-4">
```
Comment (cedricgautier) [inline mobile/src/components/Header.jsx]: Why call it brown in "bg-brown-100" and not primary color when it's one of our main colors ?

File: mobile/src/components/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    <Text className="text-lightPink-100 text-4xl font-extrabold">ÀIRNEIS</Text>
```
Comment (cedricgautier) [inline mobile/src/components/Header.jsx]: Same as above for pink

File: mobile/src/components/Header.jsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    <View className="flex flex-row items-center">
+      <ButtonWithSvg>
+        <MagnifyingGlassIcon {...mobileConfig.icons.lgIcon} color={colors.lightPink[100]} />
```
Comment (cedricgautier) [inline mobile/src/components/Header.jsx:12]: Same as above, don't use "lightPink". Thanks

File: mobile/src/components/Header.jsx [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+      </ButtonWithSvg>
+      <ButtonWithSvg>
+        <ShoppingCartIcon {...mobileConfig.icons.lgIcon} color={colors.lightPink[100]} style={{ marginLeft: 8 }} />
+      </ButtonWithSvg>
+      <ButtonWithSvg>
+        <Bars3Icon {...mobileConfig.icons.xlIcon} color={colors.lightPink[100]} style={{ marginLeft: 8 }} />
```
Comment (cedricgautier) [inline mobile/src/components/Header.jsx]: - Why is there a margin left style ? 
- Can you also remove "lightPink"

File: mobile/src/components/Page.jsx [hunk 8]
```diff
@@ -0,0 +1,16 @@
+import { SafeAreaView, ScrollView, View } from "react-native"
+import Header from "./Header"
+
+const Page = props => {
+  const { children } = props
+
+  return (
+    <View className="h-screen flex flex-col w-full">
+      <SafeAreaView className="bg-[#382E2B]"></SafeAreaView>
```
Comment (cedricgautier) [inline mobile/src/components/Page.jsx]: Don't use magic strings!

File: mobile/src/components/Page.jsx [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+      <Header />
+      <ScrollView className="grow overflow-y-auto flex flex-col w-full bg-[#FFEDE6]">{children}</ScrollView>
```
Comment (cedricgautier) [inline mobile/src/components/Page.jsx]: Same as above, magic strings

File: mobile/tailwind.config.cjs [hunk 10]
```diff
@@ -1,7 +1,12 @@
 module.exports = {
   content: ["./App.{js,jsx,ts,tsx}", "./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
-    extend: {}
+    extend: {
+      colors: {
+        lightPink: { 100: "#FFEDE6" },
+        brown: { 100: "#382E2B" }
```
Comment (cedricgautier) [inline mobile/tailwind.config.cjs]: Aren't these our primary, secondary colors? Can you please check with out Figma file. Thanks

Comment (hallycsse) [reply mobile/tailwind.config.cjs]: all done

**Review by KyDev25 — APPROVED** [review]
I haven't been able to test it on Android yet. But for me the code is also good.

File: mobile/src/components/Page.jsx [hunk 11]
```diff
@@ -0,0 +1,16 @@
+import { SafeAreaView, ScrollView, View } from "react-native"
+import Header from "./Header"
+
+const Page = props => {
+  const { children } = props
+
+  return (
+    <View className="h-screen flex flex-col w-full">
+      <SafeAreaView className="bg-secondary"></SafeAreaView>
```
Comment (avetisk) [inline mobile/src/components/Page.jsx:9]: Empty tags should be self-closed.

**Pull request merged**